html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Html Forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="get">
+      <h1>Coding Form!</h1>
+      <div>
+        <label>Email: <br /></label>
+        <input type="text" name="Email" />
+      </div>
+      <div>
+        <label>Why is coding cool? <br /></label>
```

Comment:
This `label` needs to be configured with a `for` attribute that matches the `id` attribute on its form control element.